README.md

```diff
@@ -0,0 +1,26 @@
+# Welcome to the HandCarry.App server code!
+
+## Requirements
+
+Command:Version
+docker-compose:>=1.24.0
+pip:>=19.1.1 
+
+## Setting it up
+
+In an appropriate directory:
+* git clone https://github.com/silinternational/handcarry-api.git
+* cd handcarry-api
+* docker-compose up -d
```

Comment:
```suggestion
* make
```

Reply:
Make file is in application directory, not the root cloned directory.